microk8s-resources/actions/values.yaml

```diff
@@ -0,0 +1,344 @@
+# Default values for Falco.

```

Comment:
Same here, create a directory named falco under actions and place the manifest there.